### DISCUSSION TITLE
Error 1015

### DISCUSSION THREAD

**Discussion by orbar1** [ultrafunkamsterdam/undetected-chromedriver#1195] [Q&A]
Hey everyone, how are you?
So recently i've been getting this "Error 1015 - You are being rate limited"
I would like to know what would be an ideal solution?
Is it just a matter of increasing my sleep period of time between each action or can I do anything else to solve it?
Thanks a bunch!

**Comment by mir3u** [comment]
You should use more ips/proxies to be able to get less rate limited. More proxies + one request each 5 seconds or so would make you less rate limited
